docs/examples.rst

```diff
@@ -53,29 +48,32 @@ Example: synthesizing 1000 soundscapes in one go
     time_stretch_min = 0.8
     time_stretch_max = 1.2
 
+    # generate a random seed for this Scaper object
+    seed = np.random.randint(0, 100000)
```

Comment:
This will give a different seed each time. I think the goal is to illustrate, in the simplest manner possible, how to get reproducible results. So I would replace this with just `seed = 123`.